### PR TITLE
Pre refactor bs remove

### PR DESCRIPTION
**Pull request by thinknoack**
Update to admin and dataset pages. on/components `page.scss` has nessisary admin scss. And on/app `pre-refactor-with-wrapper-main.scss` all the need bootstrap styling for the old imports to work without using BS.

### CODE REVIEW

File: packages/openneuro-server/src/graphql/utils/file.js [hunk 1]
```diff
@@ -29,10 +29,13 @@ export function UpdatedFile(filepath, size) {
...
-  // keep files that havent had their annex objects removed
-  return files.filter(({ key }) => !removedAnnexObjectKeys.includes(key))
-}
+export const filterRemovedAnnexObjects =
+  (datasetId, userInfo) => async files => {
+    const removedAnnexObjectKeys = (
+      await BadAnnexObject.find({ datasetId }).exec()
+    ).map(({ annexKey }) => annexKey)
+    // keep files that havent had their annex objects removed
+    return userInfo.admin
```
Comment (nellh) [inline packages/openneuro-server/src/graphql/utils/file.js]: This crashes the resolver for anonymous users.

**Review by nellh — CHANGES_REQUESTED** [review]

File: packages/openneuro-server/src/graphql/utils/file.js (hunk 1 above)
Comment (thinknoack) [reply packages/openneuro-server/src/graphql/utils/file.js]: hmm. this was something @david-nishi helped with. I was not able to see files that were "flagged" by a dataset owner as admin.

**Review by thinknoack — COMMENTED** [review]
@nellh i have added more updates to assist with the removal of the bootstrap scss.

File: packages/openneuro-server/src/graphql/utils/file.js (hunk 1 above)
Comment (thinknoack) [reply packages/openneuro-server/src/graphql/utils/file.js]: @david-nishi pushed a fix

**Review by nellh — APPROVED** [review]
Just one simple snapshot test failure.

**Pull request merged**